src/shared/styles.js

```diff
@@ -9,7 +9,7 @@ export const color = {
    lighter: '#F8F8F8',
    light: '#F3F3F3',
    mediumlight: '#EEEEEE',
-   medium: '#DDDDDD',
+   medium: '#999999',
```

Comment:
As long as we have `mediumdark` that has a value of `#999999` I would restore `medium` value to `#DDDDDD` to keep the scale consistent.